scripts/typescript_checks.py

```diff
@@ -44,6 +798,8 @@
 COMPILED_JS_DIR = os.path.join('local_compiled_js_for_test', '')
 TSCONFIG_FILEPATH = 'tsconfig.json'
 STRICT_TSCONFIG_FILEPATH = 'tsconfig-strict.json'
+MOCK_STRICT_TSCONFIG_FILEPATH = 'mock-tsconfig-strict.json'
```

Comment:
I don't think "mock" is quite right here since this isn't a test. Maybe "temp" so that if it somehow doesn't get cleaned up, people know it's a dynamically generated file?

Reply:
Done